public/javascripts/client.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready( () => {
+  const modal = document.querySelector( '.modal' )
```

Comment:
This set up is confusing here since it's only used in very specific locations, below.  If these items are not re-used, I would just use them directly inline instead of creating a temporary variable.